drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -298,6 +298,7 @@ public void onRunScenario(List<Integer> indexOfScenarioToRun) {
                         simulation::getScenarioByIndex
                 )
         );
+        view.showBusyIndicator(ScenarioSimulationEditorConstants.INSTANCE.running());
```

Comment:
@danielezonca 
Please use this one
**CustomBusyPopup**  (see firs line inside _loadContent_ method) that has been implemented specifically for ScenarioSimulation